WireDomain/Tests/WireDomainTests/Repositories/UserRepositoryTests.swift

```diff
@@ -44,7 +44,8 @@ class UserRepositoryTests: XCTestCase {
         usersAPI = MockUsersAPI()
         sut = UserRepository(
             context: context,
-            usersAPI: usersAPI
+            usersAPI: usersAPI,
+            isFederationEnabled: true
```

Comment:
do we need an extra test for isFederationEnabled: false?

Reply:
done